### ISSUE TITLE
ND4J: ReadTxt Issue

### ISSUE THREAD

**Issue by boing102** [deeplearning4j/deeplearning4j#6632]
Hi,

I came across unexpected and incorrect behaviour of readTxt function. I have a file:
```
{
"filefrom:" "dl4j",
"ordering:" "c",
"shape":	[6, 3, 1],
"data":
[[0.00 ,   0.00 ,   0.00] ,   

 [0.00 ,   0.00 ,   0.00] ,   

 [420.00 ,   420.00 ,   420.00] ,   

 [420.00 ,   420.00 ,   420.00] ,   

 [0.00 ,   0.00 ,   0.00] ,   

 [0.00 ,   0.00 ,   0.00]]
}
```

When I print NDArray initiated by using the readTxt function it returns this:

```
[[0.00,  0.00,  420.00],  

 [420.00,  0.00,  0.00],  

 [0.00,  0.00,  0.00],  

 [0.00,  0.00,  0.00],  

 [0.00,  0.00,  0.00],  

 [0.00,  0.00,  0.00]]
```

Version Information
nd4j-native-platform - 0.9.1
OS - Ubuntu 18.04.1 LTS

**Comment by AlexDBlack** [comment]
Thanks for reporting - I looked into this, and it seems the format you provide doesn't match the shape/output produced by 1.0.0-beta2. I've added a check for this case plus have added some extra tests here: https://github.com/deeplearning4j/deeplearning4j/pull/6634
It's quite possible that the format has been changed/fixed since 0.9.1.

FYI we're on 1.0.0-beta2 now - I'd suggest upgrading for a lot of bug fixes, new features and performance improvements.
